packages/mobile/src/merchantPayment/hooks.ts

```diff
@@ -64,7 +65,7 @@ export function useMerchantPayments(apiBase: string, referenceId: string) {
 
     setStatus(PaymentStatus.Pending)
     try {
-      await charge.submit({ phoneNumber: e164PhoneNumber })
+      await charge.submit({ address: { country: 'ES' } })
```

Comment:
nit: could you add a comment that it's only temporary for Celo Connect?

Reply:
Yes